packages/commercetools/composables/src/types/Locale.ts

```diff
@@ -0,0 +1,9 @@
+import { Ref } from '@vue/composition-api';
+import { LocaleItem } from '@vue-storefront/commercetools-api/lib/src/types/setup';
+
```

Comment:
We should pass only data types here and take care of Refs/computed properties typing inside the factory - like in other factories. By giving a freedom of deciding what it is we can lose possibility to modify that by ourselves in the future for all integrations in one place

Reply:
I am using the exact same types that were here before "refactoring". It requires further work from CT team anyway to fully implement it.

Reply:
Well the task was also to refactor current version of implemented comparable, the job should be done on the factory side because this is where you pass the types ;)